cdc/sink/manager/kafka/manager.go

```diff
@@ -106,27 +106,27 @@ func (m *TopicManager) tryUpdatePartitionsAndLogging(topic string, partitions in
 		m.topics.Store(topic, partitions)
 		log.Info(
 			"store topic partition number",
-			zap.String("duration", topic),
+			zap.String("topic", topic),
 			zap.Int32("partitionNumber", partitions),
 		)
 	}
 }
 
 // CreateTopic creates a topic with the given name.
```

Comment:
Can you add a comment to explain the meaning of the return `int32` value?

Reply:
Good catch! Done!